### ISSUE TITLE
`My favorites` displays `99+` instead of the actual number when it exceed 99

### ISSUE THREAD

**Issue by amalgame21** [nextcloud/maps#1292]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Is your feature request related to a problem? Please describe.**
`My favorites` displays `99+` instead of the actual number when it exceed 99
I need the actual number count of the favorites points

**Describe the solution you'd like**
`My favorites` displays `99+` instead of the actual number when it exceed 99

**Describe alternatives you've considered**
Make it to `999+` or `9999+` if the UI is the problem, `99+` is too small for normal use case.
Or let users to know the actual counts in the 3-dot menu, then keep displaying `99+` in `My favorites` is fine.

**Additional context**

**Comment by ant0nwax** [comment]
did you take into concideration that instead of this is a bug it would be a feature request as this could be a design implementation? there are people who are happy with the 99+

**Comment by amalgame21** [comment]
> did you take into concideration that instead of this is a bug it would be a feature request as this could be a design implementation? there are people who are happy with the 99+

Yes, so I mentioned in the alternatives, like keeping the 99+ and let us know the actual count somewhere.

I kept manually editing the `.favorites_share.json` files because of the bug in https://github.com/nextcloud/maps/issues/1288, in order to share the map to other people's accounts. So I wanted the actual count in case I missed some locations while manually editing.
